invenio_communities/alembic/72b37bb4119c_create_indeces_for_bucket_id.py

```diff
@@ -0,0 +1,40 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Create indeces for bucket_id."""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '72b37bb4119c'
+down_revision = '5c68d45c80f0'
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
+    """Upgrade database."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index(op.f('ix_communities_metadata_bucket_id'), 'communities_metadata', ['bucket_id'], unique=False)
+    op.drop_index('ix_uq_partial_files_object_is_head', table_name='files_object')
+    op.alter_column('files_objecttags', 'key',
+               existing_type=sa.TEXT(),
+               type_=sa.String(length=255),
+               existing_nullable=False)
```

Comment:
Looks like there is no alembic recipe for these changes, I guess we want to keep them.